crates/astria-sequencer/src/accounts/types.rs

```diff
@@ -117,6 +192,12 @@ impl From<u32> for Nonce {
     }
 }
 
+impl From<Nonce> for u32 {
```

Comment:
like turning this into just methods on `Nonce`?

Reply:
Yeah. So this would be `to_u32`.

I know I didn't do this for `U128`, but it felt justified because that's for converting between primitive-likes.

For things like `Nonce` I feel like there is more structure to justify these conversions.